.travis.yml

```diff
@@ -2,4 +2,4 @@ language: scala
 jdk:
   - oraclejdk8
 sudo: false
-script: mvn clean verify
\ No newline at end of file
+before_install: sh scripts/build-flink-ps.sh
```

Comment:
i think you rebased on the wrong file as you can see [here](https://github.com/proteus-h2020/proteus-solma/blob/master/.travis.yml).